ui/app/mainui/Popups.qml

```diff
@@ -150,20 +150,41 @@ QtObject {
         openPopup(profilePopupComponent, {publicKey: publicKey, parentPopup: parentPopup}, cb)
     }
 
+    function getContactDetails(publicKey) {
+        const details = Utils.getContactDetailsAsJson(publicKey)
+        return {
+            publicKey: publicKey,
+            localNickname: details.localNickname,
+            name: details.name,
+            displayName: details.displayName,
+            alias: details.alias,
+            ensVerified: details.ensVerified,
+            onlineStatus: details.onlineStatus,
+            largeImage: details.largeImage,
+            isContact: details.isContact,
+            trustStatus: details.trustStatus,
+            isBlocked: details.isBlocked
+        }
+    }
+
     function openNicknamePopup(publicKey: string, contactDetails, cb) {
```

Comment:
We don't need the `contactDetails` anymore since you get those from the Utils function.

However, since we'll be moving in the direction to no longer have the `asJson` functions, I wonder if we're not just making the future refactors hard, because we'll still have to get the details from somewhere?

If we say that `Popups.qml` will have access to the store of contacts, then this is all fine.

CC'ing @alexjba since he's the one that worked on the contacts refactor.

Reply:
the reason I didn't' remove from this and other methods is to contain the refactor into stages, since there are tons of other code calling methods like this all over the place